src/course/en/course.json

```diff
@@ -54,7 +52,8 @@
                 "ariaRegion": "This component is a multiple choice question. Once you have selected an option select the submit button below."
             },
             "_media": {
-                "ariaRegion": "This is a media component which plays a video. Select the play / pause button to watch or listen. Alternatively you can select the link below for the transcript."
+                "ariaRegion": "This is a media component which plays a video. Select the play / pause button to watch or listen. Alternatively you can select the link below for the transcript.",
+                "transcriptButton": "Alternatively you can select the link below for the transcript."
```

Comment:
In this case, it looks like it got separated out [here](https://github.com/adaptlearning/adapt_framework/commit/8d56f9cdb108faa4158b0cedef64b0d990979b29), but it crept back in [here](https://github.com/adaptlearning/adapt_framework/commit/a39dcc6f1386aa6aa473677d4d00b3a325a2f4d7). I'll remove it again to match the schema unless anyone has any objections.